Formula/sshguard.rb

```diff
@@ -70,6 +75,6 @@ def plist; <<-EOS.undent
   end
 
   test do
-    assert_match version.to_s, shell_output("#{sbin}/sshguard -v 2>&1", 78)
+    assert_match "SSHGuard #{version.to_s}", shell_output("#{sbin}/sshguard -v 2>&1", 0)
```

Comment:
`0` is assumed with `shell_output`. Only need to write it explicitly if it isn't `0`.